static/src/javascripts/projects/commercial/modules/third-party-tags/doubleclick-ad-free.js

```diff
@@ -1,20 +0,0 @@
-// @flow
-
-import { commercialFeatures } from 'common/modules/commercial/commercial-features';
-import config from 'lib/config';
-
-const doubleClickRandom = (): string => {
-    const axel = Math.random();
-    const a = axel * 10000000000000;
-    return a.toString();
-};
-
-export const doubleClickAdFree: ThirdPartyTag = {
-    shouldRun:
-        commercialFeatures.adFree &&
```

Comment:
so we used to put ad free users in a special bucket for ad free identified by the random number, but now we don't show ads for anyone so this isnt' needed any more?

Reply:
I mean for youtube specifically as this seems to reference a youtube switch

Reply:
Yep, exactly that!